dashboard/templates/dashboard/portal.html

```diff
@@ -381,7 +381,7 @@ <h3>Academic</h3>
         <h3>Logistics</h3>
         <ul>
           <li>
-            <a href="https://us5.campaign-archive.com/home/?u=5c818a0be2f35ee90344a471e&id=808c50e8c9">Announcement emails</a>
+            <a href="https://otis.evanchen.cc/dash/announce/">Announcement emails</a>
```

Comment:
can you revert this? because (1) this should be in a separate commit, (2) we should use the dynamic Django URL rather than hard-coding it.

i'll address this in 369677529226be7b9f266472523699e4a4a0c0b2